command_test.go

```diff
@@ -2253,14 +2253,70 @@ func TestTraverseWithParentFlags(t *testing.T) {
 	if err != nil {
 		t.Errorf("Unexpected error: %v", err)
 	}
-	if len(args) != 1 && args[0] != "--add" {
+	if len(args) != 1 || args[0] != "--int" {
```

Comment:
The original condition seems wrong. `--add` wasn't mentioned anywhere else in the repo and from how I understand the test, `||` makes sense here, not `&&`.